app/services/preserved_object_handler.rb

```diff
@@ -32,29 +32,32 @@ def initialize(druid, incoming_version, incoming_size, endpoint)
     @logger = PreservationCatalog::Application.logger
   end
 
-  def create_after_validation
+  def create_after_validation(checksums_validated = false)
     results.check_name = 'create_after_validation'
     if invalid?
       results.add_result(AuditResults::INVALID_ARGUMENTS, errors.full_messages)
     elsif PreservedObject.exists?(druid: druid)
       results.add_result(AuditResults::DB_OBJ_ALREADY_EXISTS, 'PreservedObject')
     elsif moab_validation_errors.empty?
-      create_db_objects(PreservedCopy::VALIDITY_UNKNOWN_STATUS)
+      creation_status = (checksums_validated ? PreservedCopy::OK_STATUS : PreservedCopy::VALIDITY_UNKNOWN_STATUS)
+      create_db_objects(creation_status, checksums_validated)
     else
-      create_db_objects(PreservedCopy::INVALID_MOAB_STATUS)
+      create_db_objects(PreservedCopy::INVALID_MOAB_STATUS, checksums_validated)
     end
 
     results.report_results
   end
 
-  def create
+  def create(checksums_validated = false)
     results.check_name = 'create'
     if invalid?
       results.add_result(AuditResults::INVALID_ARGUMENTS, errors.full_messages)
     elsif PreservedObject.exists?(druid: druid)
       results.add_result(AuditResults::DB_OBJ_ALREADY_EXISTS, 'PreservedObject')
     else
-      create_db_objects(PreservedCopy::VALIDITY_UNKNOWN_STATUS)
+      creation_status = (checksums_validated ? PreservedCopy::OK_STATUS : PreservedCopy::VALIDITY_UNKNOWN_STATUS)
+      ran_moab_validation! if checksums_validated # set validation timestamps
```

Comment:
why do we set `ran_moab_validation = true` here? if `checksum_validated` returns true? ... does `checksum_validation` run `moab_validation`? i may be confused

Reply:
ah makes sense, thanks